src/dialect/postgres/postgres-introspector.ts

```diff
@@ -70,7 +70,7 @@ export class PostgresIntrospector implements DatabaseIntrospector {
           .as('auto_incrementing'),
       ])
       // r == normal table
-      .where('c.relkind', 'in', ['r', 'v'])
+      .where(qb => qb.where('c.relkind', '=', 'r').orWhere('c.relkind', '=', 'v'))
```

Comment:
`.orWhere` is deprecated. Let's use the `ExpressionBuilder` instead:

```suggestion
      .where(({ cmpr, or }) => or([
        cmpr('c.relkind', '=', 'r'),
        cmpr('c.relkind', '=', 'v'),
      ])
```